app/models/dfe_subject.rb

```diff
@@ -1,4 +1,3 @@
-  # TODO: chase up FINANCIAL_SUPPORT
```

Comment:
I think this one
- remove it entirely and double check that there is the 2020 values
- or just leave it

@dankmitchell @misaka was there a need to keep historical data?

Reply:
There was some talk about potentially exposing historical course info? Guess we just need to make a call on that.

Reply:
TLDR; SHIP IT 🚢

I purpose we move this data into the new "Finacial incentives" database table as I think we want to retain historical data however that's out of scope for this PR and as it's in source control we can pull it back whenever we need to 👌

Reply:
Going to do this in a follow up PR 👍